views/templates/email/waiting_for_payment_adyen.html

```diff
@@ -0,0 +1,804 @@
+                <!--[if mso | IE]>
+                <tr>
+                    <td
+                            class="" width="604px"
+                    >
+
+                        <table
+                                align="center" border="0" cellpadding="0" cellspacing="0" class="" style="width:604px;" width="604"
+                        >
+                            <tr>
+                                <td style="line-height:0px;font-size:0px;mso-line-height-rule:exactly;">
+                <![endif]-->
+                <div style="Margin:0px auto;max-width:604px;">
+                    <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;" width="100%"><tbody><tr>
+                        <td style="border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; direction: ltr; font-size: 0px; padding: 15px 50px 40px; text-align: center; vertical-align: top;" align="center">
+                            <!--[if mso | IE]>
+                            <table role="presentation" border="0" cellpadding="0" cellspacing="0">
+
+                                <tr>
+
+                                    <td
+                                            class="" style="vertical-align:top;width:504px;"
+                                    >
+                            <![endif]-->
+                            <div class="mj-column-per-100 outlook-group-fix" style="font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;" align="left" width="100%">
+                                <table border="0" cellpadding="0" cellspacing="0" role="presentation" width="100%" style="border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt;"><tbody><tr>
+                                    <td style="border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; background-color: #fefefe; border: 1px solid #DFDFDF; vertical-align: top; padding: 10px 0;" bgcolor="#fefefe">
+                                        <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt;" width="100%"><tr>
+                                            <td align="left" style="border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; font-size: 0px; padding: 10px 25px; word-break: break-word;">
+                                                <div style="font-family:Open sans, arial, sans-serif;font-size:14px;line-height:25px;text-align:left;color:#363A41;" align="left">
+                                                    Your order with the reference <span class="label" style="font-weight: 700;">{order_name}</span> has been placed successfully. You can expect <span class="label" style="font-weight: 700;">delivery as soon as your payment is received</span>.
```

Comment:
Is this copied from a PS template? Not sure if the font-weight styling makes sense here:

Your order with the reference **order_name** has been placed successfully. You can expect **delivery as soon as your payment is received**.

Should it be maybe:

Your order with the reference **order_name** has been placed successfully. You can expect the delivery as soon as your **payment is received**.

Reply:
@acampos1916 Yes the structure and styling was copied from the PrestaShop template used on a pending bank transfer